lib/tennis_set.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative 'tennis_game'
+require_relative 'tennis_score'
+
+# defines a tennis set, and handles all scoring for the game
+class TennisSet < TennisScore
+  def initialize(skip_games, players)
+    super(6, 2, 'set', players)
```

Comment:
I think this explanation is sufficient then in a normal tennis match you'd have tie breaks etc except in the last set - but that is definitely out of scope for this exercise!